tiered-storage/jcloud/src/main/java/org/apache/bookkeeper/mledger/offload/jcloud/impl/BlobStoreManagedLedgerOffloader.java

```diff
@@ -218,12 +228,19 @@ public String getOffloadDriverName() {
                         }
                         entryBytesWritten += blockStream.getBlockEntryBytesCount();
                         partId++;
+                        this.mxBean.recordOffloadBytes(extraMetadata.get(MANAGED_LEDGER_NAME),
+                                blockStream.getBlockEntryBytesCount());
                     }
 
                     dataObjectLength += blockSize;
                 }
 
+                long startUploadTime = System.nanoTime();
                 writeBlobStore.completeMultipartUpload(mpu, parts);
+                this.mxBean.recordWriteToStorageLatency(extraMetadata.get(MANAGED_LEDGER_NAME),
```

Comment:
Each part has its own upload time, right? I'm not sure they are async way or sync way, and how to describe this duration.